spec/spec_helper.rb

```diff
@@ -177,4 +177,21 @@ def main_app
   config.include FactoryGirl::Syntax::Methods
 
   config.infer_spec_type_from_file_location!
+
+  config.expect_with :rspec do |expectations|
+    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+  end
+
+  config.default_formatter = 'doc' if config.files_to_run.one?
+
+  config.order = :random
+  Kernel.srand config.seed
+
+  config.shared_context_metadata_behavior = :apply_to_host_groups
+
+  config.filter_run_when_matching :focus
+
+  config.example_status_persistence_file_path = 'spec/examples.txt'
```

Comment:
We could, although this is the default path from upstream, so I assume this is where people would expect to find the file.